src/Navigation/Drawer.tsx

```diff
@@ -36,15 +36,17 @@ const Drawer = ({ username }: { username: string }) => {
   return (
     <div className="drawer lg:drawer-open">
       <input id="my-drawer-2" type="checkbox" className="drawer-toggle" />
-      <div className="drawer-content flex flex-col items-center justify-center lg:relative">
-        {username && (
+      <div
+        className={`drawer-content flex flex-col items-center justify-center ${!username && "h-lvh"} lg:relative`}
```

Comment:
would the drawer-content div not be position: relative by default? In regards to the ` lg:relative `?

Reply:
Just read that position: static is the default so you can ignore the previous comment 😅 my bad